pipeline-steps/maas.groovy

```diff
@@ -0,0 +1,67 @@
+def prepare(Map args) {
+  common.conditionalStage(
+    stage_name: 'Create MaaS Entities',
+    stage: {
+      withCredentials([
+        string(
+          credentialsId: "dev_pubcloud_username",
+          variable: "PUBCLOUD_USERNAME"
+        ),
+        string(
+          credentialsId: "dev_pubcloud_api_key",
+          variable: "PUBCLOUD_API_KEY"
+        ),
+        string(
+          credentialsId: "dev_pubcloud_tenant_id",
+          variable: "PUBCLOUD_TENANT_ID"
+        )
+      ]){
+        dir("rpc-gating/playbooks"){
+          pyrax_cfg = common.writePyraxCfg(
+            username: env.PUBCLOUD_USERNAME,
+            api_key: env.PUBCLOUD_API_KEY
+          )
+          withEnv(["RAX_CREDS_FILE=${pyrax_cfg}"]){
+            common.venvPlaybook(
+              playbooks: ['aio-maas-entities.yml'],
+              venv: ".venv",
+              args: [
+                "-e server_name=\"${args.instance_name}\""
+              ],
+              vars: args
+            )
+          } // withEnv
+        } // dir
+      } // withCredentials
+    } // stage
+  ) // conditionalStage
+}
+
+def deploy() {
+  common.overwriteConfig(
```

Comment:
I don't see this method in common.groovy. Is it supposed to be `prepareConfigs()`? Also, it may make more sense to have this method call moved into either the prepare method of this groovy script (and rename Create MaaS Entities to Prepare MaaS) or into the `Prepare RPC Configs` stage of multi_node_aio_prepare.groovy.